### ISSUE TITLE
Maximum number of supported shared content streams

### ISSUE THREAD

**Issue by hassanctech** [aws/amazon-chime-sdk-js#2164]
### What are you trying to do?
What is the maximum number of shared content streams that are supported?  

Under scaling limitations this (https://aws.amazon.com/chime/chime-sdk/faq/) suggests 2 per session.  Is that a hard limit?

### How can the documentation be improved to help your use case?

*For example: Include a section on how to configure the meeting session.*

### What documentation have you looked at so far?

*For example: README.md, Amazon Chime SDK Developer Guide, etc.

**Comment by ltrung** [comment]
Yes 2 is the hard limit right now for content share.

**Comment by hassanctech** [comment]
Yes we have a use case in which a user may only share one screen/window (or on the sender side we can package the multiple streams into one), but up to 5 different users may need to share streams, so what would it take on your end to up that limit to 5?

**Comment by xuesichao** [comment]
Thank you for submitting a feature request. We will track your request via an entry on the [Amazon Chime SDK Project Board](https://github.com/orgs/aws/projects/253) for centralized visibility of the request and its status and close this GitHub issue.
Thank you for your feedback and for contributing to the future of the service.
